Assets/Scripts/Enemies/Bosses/SpikeBall.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UnityEngine;
+
+public class SpikeBall : EnemyMovement
```

Comment:
An integration test for this would be nice. I would test to make sure the ball moves straight, that it has the correct amount of spikes, and at some point the spikes move straight according to the way they are pointing.